kolibri/plugins/learn/assets/src/actions.js

```diff
@@ -36,7 +36,35 @@ const fetchNodes = ({ dispatch }, id) => {
   });
 };
 
+
+const searchNodes = ({ dispatch }, params, page) => {
+  // Get the collection from ContentNodeResource.
+  const contentCollection = Kolibri.resources.ContentNodeResource.getCollection();
+  const wantedFields = ['pk', 'title', 'kind', 'instance_id', 'content_id', 'description', 'files'];
```

Comment:
However if we reduce `wantedFields` down to this:
`const wantedFields = ['pk', 'title', 'kind', 'description'];`

We reduce the payload by a lot more:
![image](https://cloud.githubusercontent.com/assets/1680573/16671835/bd787978-4456-11e6-8db3-03b2d956aa64.png)